### DISCUSSION TITLE
ender 3 pro 4.2.7

### DISCUSSION THREAD

**Discussion by clark0072020** [Jyers/Marlin#2094] [Q&A]
ive recently bought the runout fillament sensor and the autobed levelling probe  ive installed both components, when i tried to do a firmware upgrade its left me with a blue screen? any help please?

**Comment by mautty** [comment]
The blue screen is expected, you have to use the web interface for control of the printer from now on

**Reply by JPT77** [reply]
No? 
or is this GIT a special version of marlin?
usually you get a blue screen while it is updating the firmware from sd card. 
It should reboot after a few seconds. 
maybe something wrong with the firmware?
did you try another file?
